### PR TITLE
# Add section on Superscript, Subscript and Special Symbols

### PR DESCRIPTION
**Pull request by sspencerwire**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by sspencerwire** [comment]
@Grammaresque can you look this over and see if it needs any tweaking?

File: docs/guides/contribute/rockydocs_formatting.md [hunk 1]
```diff
@@ -291,6 +291,57 @@ And here's how that renders when displayed:
...
 
+## Superscript, Subscript and Special Symbols
+
+Superscript and subscript notation are not normal markdown, but are supported in Rocky Linux documentation via the HTML tags used for the same purpose. Superscript places text entered between the tags slightly above the normal text, while subscript places the text slightly below. Superscript is by far the most commonly used of these two in writing. Some special characters already appear in superscript without adding the tags, but you can also combine the tag to change the orientation of those characters as seen with the copyright symbol below. You can use superscript to:
+
+* represent ordinal numbers, such as 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>
+* copyright and trademark symbols, like <sup>&copy;</sup>, <sup>TM</sup> or &trade;, &reg;
+* as notation for references, such as this<sup>1</sup>, this<sup>2</sup> and this<sup>3</sup>
```
Comment (Grammaresque) [inline docs/guides/contribute/rockydocs_formatting.md:300]: add comma after second item in list.

**Review by Grammaresque — COMMENTED** [review]
Looks good. Seems pretty straightforward to me.

File: docs/guides/contribute/rockydocs_formatting.md (hunk 1 above)
Comment (sspencerwire) [reply docs/guides/contribute/rockydocs_formatting.md:300]: Technically, it's the same item. TM or $trade; They both stand for a trademark but the second renders more closely to the actual trademark symbol. That said, I'll add a comma. ;-)

Comment (sspencerwire) [reply docs/guides/contribute/rockydocs_formatting.md:300]: Thank you for your review!!!

**Pull request merged**